scripts/run_integration_tests.sh

```diff
@@ -88,16 +88,14 @@ if [ ! -f "$VMLINUX_IMAGE" ]; then
 fi
 
 VIRTIOFSD="$WORKLOADS_DIR/virtiofsd"
```

Comment:
Just a nit, but could you change the commit message title prefix `src:` into `ci:`?